tools/test_history.py

```diff
@@ -82,7 +82,7 @@ def make_columns(
         if job in omitted:
             total_omitted += omitted[job]
     if total_omitted > 0:
-        columns.append(f'({total_omitted} S3 reports omitted)')
+        columns.append(f'({total_omitted} job re-runs omitted)')
```

Comment:
do we know if it is the first run that gets displayed or the last one? (and can we control that?)

Reply:
I'm not entirely sure